app/views/cards/edit.html.erb

```diff
@@ -1 +1,76 @@
-Edit Card View (see deliverable doc)
+<div class="page page-dashboard">
+  <div class="wrapper wrapper--light">
+    <div class="container">
+      <div class="pagehead">
+        <div class="row">
+          <div class="col-sm-6">
+            <ol class="breadcrumb">
+              <li><%= link_to 'Decks', decks_path %></li>
+              <li><%= link_to @deck.title, deck_path(@deck) %></li>
+              <li class="active">Card <%= @card.id %></li>
+            </ol>
```

Comment:
the `breadcrumb` should really be a partial that is able to dynamically generate the necessary html. it might need a helper to extract the logic away from the view, but this current strategy will quickly become unwieldy to maintain